spec/helpers/locations_helper_spec.rb

```diff
@@ -51,7 +51,7 @@
     let(:location) { FactoryBot.create(:location, startdate: nil) }
 
     before do
-      Timecop.freeze(2023, 10, 10)
+      allow(Time).to receive(:now).and_return(Time.zone.local(2023, 10, 10))
       allow(FormHandler.instance).to receive(:lettings_in_crossover_period?).and_return(true)
```

Comment:
if we're removing the freeze before can we remove the unfreeze after?

Reply:
Yup!